platform-coop-toolkit/coop-library
# feat: update Pinecone to alpha 14

## Description

Update Pinecone version to Alpha 14, updated branding, added global navigation

To add global navigation, in the dashboard, add a new menu "Global Navigation" with a link to https://platform.coop/ as the first menu item, and a link to the Front Page as the second item. In menu locations, add this new "Global Navigation" menu to the Global Navigation location.

## Steps to test

1. In theme directory run `npm i`
2. Load Resource Library Wordpress site
3. Confirm that styles look as expected (buttons and form elements), branding logos look correct, and global navigation persists across all pages.

**Expected behavior:**

Nothing should look broken.

## Additional information

There are some known issues which have been filed already in Pinecone.
https://github.com/platform-coop-toolkit/pinecone/issues
